File: Models/Igrac.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Models
{
    [Table("Igraci")]
    public class Igrac
    {
        [Key]
        public int ID { get; set; }

        [Required]
        [MaxLength(30)]
        public string Ime { get; set; }

        [Required]
        [MaxLength(30)]
        public string Prezime { get; set; }

        [Required]
        [Range(1, 99)]
        public int BrojDresa { get; set; }

        [Required]
        [Range(16, 42)]
        public int BrojGodina { get; set; }

        [Required]
        [Range(1, 5)]
        public int Kvalitet { get; set; }

        [Required]
        public Pozicija Pozicija { get; set; }

        [Required]
        public Nacionalnost Nacionalnost { get; set; }

        public virtual TimFC Tim { get; set; }
    }
}

File: FrontEnd/Igrac.js
export class Igrac
{
    constructor(Ime, Prezime, BrojDresa, BrojGodina, Kvalitet, NazivPozicije, Drzavljanstvo)
    {
        this.Ime = Ime;
        this.Prezime = Prezime;
        this.BrojDresa = BrojDresa;
        this.BrojGodina = BrojGodina;
        this.Kvalitet = Kvalitet;
        this.NazivPozicije = NazivPozicije;
        this.Drzavljanstvo = Drzavljanstvo;
    }
        
    PopuniTabelu(tr)
    {
        var td = document.createElement("td");
        td.innerHTML = this.BrojDresa;
        tr.appendChild(td);

        td = document.createElement("td");
        td.innerHTML = this.Ime;
        tr.appendChild(td);

        td = document.createElement("td");
        td.innerHTML = this.Prezime;
        tr.appendChild(td);

        td = document.createElement("td");
        td.innerHTML = this.BrojGodina;
        tr.appendChild(td);

        td = document.createElement("td");
        td.innerHTML = this.Kvalitet;
        tr.appendChild(td);
    }
}